scripts/go_fmt.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if [ "$(gofmt -s -l . | wc -l)" -gt 0 ]; then
+    gofmt -d .
```

Comment:
We can do this on `./...` (instead of just `.`) to also test all subdirectories, once they exist.

Reply:
Can we also do that for the line above? :smile:  (in the `if` statement? )

Reply:
Resolved this suggestion:
Small mistake between the usage of `go fmt` && `gofmt`